sjb/config/test_data/gce-artifacts.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -o xtrace
+
+base_artifact_dir="/tmp/artifacts"
+
+for instance in $( gcloud compute instances list --regexp ".*${INSTANCE_PREFIX}.*" --uri ); do
+    info="$( mktemp )"
+    gcloud compute instances describe "${instance}" > "${info}"
+    name="$( jq '.name' --raw-output "${info}" | tail -c 5 )"
+    if jq '.tags.items | contains("ocp-master")' --exit-status; then
```

Comment:
the "ocp-" prefix is configurable but we are unlikely to change it in the near term.  But we might in the future if we do an origin specific one.